src/tests/ftest/nvme/enospace.py

```diff
@@ -106,15 +197,52 @@ def get_pool_usage(self, pool_space):
 
         return pool_usage
 
-    def display_pool_stats(self, pool_space, pool_space_metrics):
-        """Display statistics on pool usage.
+    def display_table(self, title, table, cols_size, align_idx):
```

Comment:
It looks like cols_size is just tracked as the max length? FYI we could do this to calculate the max length of each column without having to keep track
It's a transpose + max string length in each column
```
cols_size = [
    max(map(lambda v: len(str(v)), (row[i] for row in table)))
    for i in range(len(table))]
```

Reply:
Yep, it will add one loop over the table, but the code will be much more maintainable :-)

- [ ] Compute the columns size into the `display_table()` method

Reply:
Luckily these tables are small, but I'm also itching for an excuse to add a dependency on `numpy`...

Reply:
For this kind of table, using mumpy sounds like using a hammer to kill a fly (i.e. French proverb) ;-)

Reply:
Yeah, it doesn't need it :( but if it was massive ;)

Reply:
> Yep, it will add one loop over the table, but the code will be much more maintainable :-)
> 
> * [x]  Compute the columns size into the `display_table()` method

Fixed with commit 6445498a45098e2fe8798e92be15d8bae3789ba7

Reply:
Thanks. And woops noticed a bug in mine that you corrected :)